documentation/3.4/content/release-notes.md

```diff
@@ -8,6 +8,7 @@ draft: false
 
 | Date               | Version  | Introduces backward incompatibilities?                                                                                                      | Change - See also, [Change log](#change-log).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 |--------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| October 26, 2022   | v3.4.4   | no                                                                                                                                          | Support added to specify resource requests and limits for Monitoring Exporter sidecar containers. This release is compatible with running in the same Kubernetes cluster as additional operators from the upcoming 4.0 release.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
Sorry but I'm having a hard time understanding this sentence, "This release is compatible with running in the same Kubernetes cluster as additional operators from the upcoming 4.0 release." Maybe slightly better, "This release of the operator (OR Operator v3.4.4) is compatible with running in the same Kubernetes cluster as additional operators from the upcoming 4.0 release."